unity3d/Assets/Swrve/Editor/SwrveCommonBuildComponent.cs

```diff
@@ -43,7 +43,7 @@ protected static void BuildAndroid (string fileName, BuildOptions opt, string[]
         UnityEngine.Debug.Log ("[####] Building " + fileName);
         EditorUserBuildSettings.SwitchActiveBuildTarget (BuildTarget.Android);
         PlayerSettings.bundleIdentifier = packageName;
-        SwrveBuildComponent.CorrectApplicationId ();
+        SwrveBuildComponent.AndroidPreBuild ();
```

Comment:
No, it's marked as obselete / deprecated, but it's not removed.  I've also made a note of it with Shelly.